os/include/tinyara/binary_manager.h

```diff
@@ -131,7 +128,7 @@ typedef struct binary_header_s binary_header_t;
 struct binary_update_info_s {
 	int available_size;
 	char name[BIN_NAME_MAX];
-	uint32_t version;
+	double version;
```

Comment:
This is not kernel version you mentioned in title. Could you rewrite the title or could you split the commit?

Reply:
I agree, I think we can have two commits based on kernel version in binary/kernel table